.travis.yml

```diff
@@ -6,12 +6,17 @@ python:
 - 3.4
 - 3.5
 - 3.6
+- 3.7
+- 3.8
+- 3.9
```

Comment:
Could you add those versions to _tox.ini_ as well?

Reply:
Absolutely!
Should I rebase my commits or are multiple commits fine?

Reply:
Multiple commits are fine!